glide.lock

```diff
@@ -1,13 +1,22 @@
-hash: 6b127577a871ff8e66cd84563cd19e9b2c339a81c9e27adc37808dab3c8a1d5a
-updated: 2016-11-14T09:22:05.45573152-08:00
+hash: 69df4410f726670e69ae64ae30dbd2b1c0c5594dda5ca732c15266f852602dfa
```

Comment:
the glide.yaml file doesn't appear to have been updated but there are tons of changes in the glide.lock file.  Was some part of our process broken before resulting in our dependencies not being captured in the glide.lock file?